src/Game/AccountManagement.java

```diff
@@ -271,20 +530,422 @@ public static void panel_welcome_hook(int n) {
+	            Panel.addButtonBackTo(Client.panelRecoveryQuestions,370, var1_1, 80, 30);
+	            Panel.addCenterTextTo(
+	                    Client.panelRecoveryQuestions,370, var1_1 - 7, "Different", 1, true);
+	            Panel.addCenterTextTo(
+	                    Client.panelRecoveryQuestions,370, var1_1 + 7, "Question", 1, true);
+	            Client.controlQuestion[idx] = Panel.addButtonTo(Client.panelRecoveryQuestions,370, var1_1, 80, 30);
+	            Panel.addButtonBackTo(Client.panelRecoveryQuestions,455, var1_1, 80, 30);
+	            Panel.addCenterTextTo(
+	                    Client.panelRecoveryQuestions,455, var1_1 - 7, "Enter own", 1, true);
+	            Panel.addCenterTextTo(
+	                    Client.panelRecoveryQuestions,455, var1_1 + 7, "Question", 1, true);
+	            Client.controlCustomQuestion[idx] = Panel.addButtonTo(Client.panelRecoveryQuestions,455, var1_1, 80, 30);
+	            var1_1 += 35;
+	        }
+	        Panel.setFocus(Client.panelRecoveryQuestions, Client.controlAnswerInput[0]);
+	        var1_1 += 10;
+	        Panel.addButtonBackTo(Client.panelRecoveryQuestions,256, var1_1, 250, 30);
+	        Panel.addCenterTextTo(
+                    Client.panelRecoveryQuestions,256, var1_1, "Click here when finished", 4, true);
+	        Client.chooseFinishSetRecoveryButton = Panel.addButtonTo(Client.panelRecoveryQuestions,256, var1_1, 250, 30);
+
+	  } catch (Exception e) {
+		  e.printStackTrace();
+	  }
+  }
+  
+  public static void create_contact_details() {
+	  try {
+		  Client.panelContactDetails = Panel.createPanel(100);
+	        int n = 256;
+	        int n2 = 400;
+	        int n3 = 25;
+	        Client.controlContactDetails =
+	                Panel.addCenterTextTo(
+	                    Client.panelContactDetails,256, n3, "@yel@Please supply your contact details", 5, true);
+	        n3 += 30;
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,256, n3, "We need this information to provide an efficient customer support service ", 1, true);
+	        n3 += 15;
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,256, n3, "and also to work out where to locate future RuneScape servers.", 1, true);
+	        n3 += 25;
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,256, n3, "We know some people are concerned about entering their email address on", 1, true);
+	        n3 += 15;
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,255, n3, "websites, and for this reason we take our users privacy very seriously.", 1, true);
+	        n3 += 15;
+	        Panel.addCenterTextTo(
+                    Client.panelContactDetails,256, n3, "For our full policy please click the relevant link below this game window", 1, true);
+	        Panel.addButtonBackTo(Client.panelContactDetails,n, n3 += 40, n2, 30);

```

Comment:
take the n3 +=40 out of here lol

Reply:
+= can happen above this line

Reply:
You moved it up too high, move the += 40 below "For our full policy"

Reply:
yep fixed now